spec/buftok_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'filewatch/buftok'
+
+describe FileWatch::BufferedTokenizer do
+
+  context "when using the default delimiter" do
+    subject { FileWatch::BufferedTokenizer.new }
+
+    it "splits the lines correctly" do
+      message = "hello\nworld\n"
+      expect(subject.extract(message)).to eq ["hello", "world"]
+    end
+  end
+
+  context "when passing a custom delimiter" do
+    subject { FileWatch::BufferedTokenizer.new("\r\n") }
```

Comment:
I understand your point but I don't want to specify a delimiter in the first example, I want a clean .new with no args. so I can't dry this.